EventKit/eventkit-cloud
# Reduce params

# Description of Improvement

Reducing the amount of parameters for export tasks since a lot of the information is already apart of the object

## How to test

Check the back end tests with `make test-back`. Everything should still work smoothly

## Quality Assurance

The following items have been updated:
